fastcrawler/test_utils_tmp.py

```diff
@@ -0,0 +1,110 @@
+from enum import StrEnum
+import re
+from typing import Callable, Dict, Tuple
+
+from fastcrawler.engine.contracts import (
+    ProxySetting,
+    RequestCycle,
+    SetCookieParam,
+    Response,
+    Url,
+    Request,
+    # EngineProto,
+)
+
+
+class Method(StrEnum):
+    GET = "GET"
+    POST = "POST"
+    PUT = "PUT"
+    DELETE = "DELETE"
```

Comment:
Add more HTTP methods, like patch and option and etc. check aiohttp library
ooh i think also inside engine we need them as well -:)